cpp/include/cudf/utilities/device.hpp

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
I don't know if we want C++ functions for these. I imagined them as just Cython functions. cc @jrhemstad

Reply:
Agreed. I think it's unnecessary to make C++ functions that just call CUDA runtime functions. Better to just call the CUDA functions from Cython.